app/assets/stylesheets/custom.css.scss

```diff
@@ -51,20 +51,20 @@
   min-height: 100%;
 
   .i-am {
-    font-size: 6em;
-    color: #033E58;
     background-color: transparent;
+    color: #033E58;
```

Comment:
Color literals like `#033E58` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#033E58` should be written as `#033e58`